library/src/test/java/com/datatorrent/lib/io/fs/AbstractFileInputOperatorTest.java

```diff
@@ -106,7 +106,7 @@ public void testSinglePartiton() throws Exception
         lines.add("f" + file + "l" + line);
       }
       allLines.addAll(lines);
-      FileUtils.write(new File(testMeta.dir, "file" + file), StringUtils.join(lines, '\n'));
+      FileUtils.write(new File(testMeta.dir + "/subdir", "file" + file), StringUtils.join(lines, '\n'));
```

Comment:
Please add more test cases considering multi level paths and files within each path.